### DISCUSSION TITLE
Problem with "-compose divide_Src -composite" construction

### DISCUSSION THREAD

**Discussion by Spica2000** [ImageMagick/ImageMagick#5586] [Help] [answered]
Hello!

Could you please help with a problem of reproducing "Divide, Divide_Dst, Divide_Src" samples [https://www.imagemagick.org/Usage/compose/#dividedst](https://www.imagemagick.org/Usage/compose/#dividedst) ?

When I run command (via cmd or via bat file in Windows 10):
`convert text_scan.png +clone -blur 0x1 +swap -compose divide_Src -composite text_divide.png`

I get almost empty white text_divide.png file, not the one shown in example. Same results (pure white file) on other files.

I've tried legacy 6.9.12-portable-Q16-x64 version as well as last 7.1

Thank you very much!

**Comment by snibgo** [comment, marked as answer]
Perhaps you are referring to this example:
```
  magick text_scan.png \( +clone -blur 0x20 \) \
          -compose Divide_Src -composite  text_scan_divide.png
```
Examples on that page use bash syntax, which uses backslash "\\" as line-continuation, and escapes parentheses "(" and ")" with backslash.

In Windows, line-continuation is caret "^", and parentheses in this context don't need to be escaped.

EDIT: I should explain that the parentheses are needed limit the burring operation to just the clone, instead of to both images. If you blur both images, and divide one by the other, the result is "1", which is white.

**Reply by Spica2000** [reply]
Thank you very very much!! The explanation is so clear,

It is very confusing for Windows users.

I should add that in last versions we should pay attention to Divide_Dst and Divide_Src commands, because in some past examples they use (+swap -divide) construction.
